### DISCUSSION TITLE
Error with Heightmaps

### DISCUSSION THREAD

**Discussion by abhi-arya1** [pokepetter/ursina#512] [Q&A]
Hi, Im trying to generate ground entities with Terrain(heightmap) as the model. 
_Ex: ground = Entity(model=Terrain(heightmap=[PATH]))_
I keep getting an error, but only with certain heightmaps, that for line 38 in __init__ for ursina\models\procedural\terrain.py, self.width=len(self.height_values) returns a TypeError: object of type 'NoneType' has no len()
Any ideas how to fix? I can provide more examples if needed. 
Thank you so much!

**Comment by pokepetter** [comment]
Make sure the heightmap is RGBA
